de-lib/src/main/java/org/iplantc/de/client/models/userSettings/UserSetting.java

```diff
@@ -0,0 +1,53 @@
+package org.iplantc.de.client.models.userSettings;
+
+import org.iplantc.de.client.models.diskResources.Folder;
+
+/**
+ * @author aramsey
+ */
+public interface UserSetting {
+
+    Boolean isEnableAnalysisEmailNotification();
+    void setEnableAnalysisEmailNotification(Boolean enable);
+
+    Boolean isEnableImportEmailNotification();
+    void setEnableImportEmailNotification(Boolean enable);
+
+    String getDefaultFileSelectorPath();
+	void setDefaultFileSelectorPath(String value);
+
+    Boolean isRememberLastPath();
+	void setRememberLastPath(Boolean remember);
+
+    Boolean isSaveSession();
+	void setSaveSession(Boolean save);
+
+    Folder getDefaultOutputFolder();
+	void setDefaultOutputFolder(Folder value);
+
+    String getDataKBShortcut();
+	void setDataKBShortcut(String value);
+
+    String getAppsKBShortcut();
+	void setAppsKBShortcut(String value);
+
+    String getAnalysisKBShortcut();
+	void setAnalysisKBShortcut(String value);
+
+    String getNotificationKBShortcut();
+	void setNotificationKBShortcut(String value);
+
+    String getCloseKBShortcut();
+	void setCloseKBShortcut(String value);
+
+    Folder getSystemDefaultOutputDir();
+	void setSystemDefaultOutputDir(Folder value);
```

Comment:
These default output setting keys were renamed by cyverse-de/terrain#8 to `default_output_folder` and `system_default_output_dir`. I think these fields will need `@Property` annotations for the new names?
If there are objections to the new name, these can be renamed back to the camel-case versions in terrain.